community/modules/compute/gke-job-template/variables.tf

```diff
@@ -113,6 +113,16 @@ variable "random_name_sufix" {
   default     = true
 }
 
+variable "persistent_volume_claim" {
```

Comment:
Need this be singular in order to match output values? It would most naturally be plural.

If so, I would probably change the input to plural and the outputs to match.

Reply:
You are correct about input-output matching. I don't have any strong feelings, just mating the pattern of a singular `network_storage`. 

I think plural is probably better because users are more likely to look at inputs than outputs.

Reply:
Updated with your suggestion.